DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -65,7 +65,9 @@ struct NetworkProtectionStatusView: View {
         })
         .applyInsetGroupedListStyle()
         .sheet(isPresented: $statusModel.showAddWidgetEducationView) {
-            widgetEducationSheet()
+            if #available(iOS 17.0, *) {
```

Comment:
This guard was missing, but the VPN education sheet should be only available in iOS 17+.